### PR TITLE
12.0 fix domain user

### PR DESCRIPTION
**Pull request by cesarlr**

### CODE REVIEW

**Comment by cesarlr** [comment]
@nelsonramirezs @max3903 Can you review it?

File: helpdesk_motive/__manifest__.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 {
     "name": "Helpdesk Motive",
-    "version": "12.0.1.0.0",
+    "version": "12.0.1.0.1",
```
Comment (dalonsod) [inline helpdesk_motive/__manifest__.py]: @cesarlr addon version shouldn't be changed, OcaBot will do it

**Review by dalonsod — CHANGES_REQUESTED** [review]
See comment, only code review at this moment (runbot is not still available).

File: helpdesk_motive/__manifest__.py (hunk 1 above)
Comment (cesarlr) [reply helpdesk_motive/__manifest__.py]: Ok! I've just removed the change

**Review by dalonsod — APPROVED** [review]
@cesarlr locally tested, LGTM.

I'm not sure if this PRs is the proper one, or if a new should be created, but maybe a good improvement IMO should be unsetting motive only if `team_id` has changed, not always. What do you think?

**Pull request closed without merging**